src/Resources/views/includes/sidebar/_profile.html.twig

```diff
@@ -35,27 +38,35 @@
             <div class="border-b">
                 <span class="inline-block whatwedo-utility-paragraph text-sm leading-none mb-2">Angemeldet als</span>
                 <div class="flex whatwedo-utility-bold text-neutral-900 mb-2">
-                    <img class="inline-block h-4 w-4 rounded-full self-center mr-2"
-                         src="{% block profile_picture %}{% endblock %}" alt=""
-                         crossorigin="anonymous" referrerpolicy="no-referrer">
-                    {% block profile_fullname %}welove@test.com{% endblock %}
+                    {% if picture_src %}
+                        <img class="inline-block h-8 w-8 md:h-10 md:w-10 rounded-full"
+                             src="{{ picture_src }}" alt=""
+                             crossorigin="anonymous" referrerpolicy="no-referrer">
+                    {% endif %}
+                    {{ name_detail ?? name }}
                 </div>
             </div>
 
+            {% if edit_link %}
+                <a
+                    href="{{ edit_link }}"
+                    class="flex whatwedo-utility-paragraph transition-colors hover:text-neutral-500"
+                >
+                    {{ bootstrap_icon('pencil-square', { class: 'h-3 w-3 self-center mr-2', alt: '' }) }}
+                    {{ 'Profil bearbeiten' }}
```

Comment:
Nit: Wrapping into a twig expression is unnecessary here. Instead of {{ 'Profil bearbeiten' }}, use 'Profil bearbeiten'.

Reply:
I updated them to use the translations